config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/ServiceAndPolicyLoaderUtil.java

```diff
@@ -38,5 +40,37 @@ public static Folder getFolder(Bundle bundle, String folderId) {
         return folderList.get(0);
     }
 
+
+    /**
+     * Append a value to end of name of the service/policy in a path if the name is found in bundle.
+     * Service/Policy Id must be different and must exist in the same folder.
+     *
+     * @param bundleEntity bundle containing original service/policy
+     * @param entity duplicate service with same path and service/policy name
+     * @return a path with a numerical value appended to the end to differentiate from original service/policy
+     */
+    public static String handleDuplicatePathName(Map<String, ? extends Folderable> bundleEntity, Folderable entity) {
+        int duplicateCounter = 2;
+        String basePath = entity.getPath();
+        String clonePath = basePath;
+
+        if (System.getProperty("exportDuplicate").equals("true")) {
```

Comment:
you can use Boolean.getBoolean("exportDuplicate")

also please add a constant for the property name, and we have to think on some better name for this
maybe "com.ca.apim.export.handleDuplicateNames", what do you think?

Reply:
let's not do com.ca.apim, instead continue using com.l7tech.*.. Rebranding is not on priority, but we should not add more to scope.. :)

Reply:
Sure sounds good. I honestly wasn't sure to name it to begin with. Thanks for the suggestion

Reply:
Going with "ca.l7tech.export.handleDuplicateNames"?

Reply:
there is no l7tech mention anywhere on the plugins Amogh, everything is com.ca.apim
I think we should stick to this.

Reply:
If that is the case, I am good with com.ca.apim.*